### PR TITLE
FSM-328:FSM-DSS changes related to top and bottom performing Ulb's

### PR DESCRIPTION
**Pull request by lakshmisravani-wtt-egov**

### CODE REVIEW

File: egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json [hunk 1]
```diff
@@ -395,7 +395,7 @@
       },
       {
         "module": "FSM",
-        "dateRefField":  "Data.@timestamp",
+        "dateRefField":  "Data.fsm.@timestamp",
         "requestQueryMap": "{\"wardId\" : \"Data.ward.name.keyword\", \r\n  \"district\" : \"Data.tenantData.city.districtCode\", \r\n\"tenantId\" : \"Data.fsm.tenantId\" \r\n}",
         "indexName": "fsm",
         "aggrQuery": "{\"aggs\":{\"AGGR\":{\"filter\":{\"bool\":{\"must_not\":[{\"term\":{\"Data.fsm.tenantId.keyword\":\"pb.testing\"}}]}},\"aggs\":{\"Total Application\":{\"value_count\":{\"field\":\"Data.@timestamp\"}}}}}}"
```
Comment (phanikiran-wtt-egov) [inline egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json:401]: Data.@timestamp should be changed to Data.fsm.@timestamp in thsi aggrQuery

File: egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json [hunk 2]
```diff
@@ -446,7 +446,7 @@
       },
       {
         "module": "FSM",
-        "dateRefField": "Data.@timestamp",
+        "dateRefField": "Data.fsm.@timestamp",
         "requestQueryMap": "{\"wardId\" : \"Data.ward.name.keyword\", \r\n  \"district\" : \"Data.tenantData.city.districtCode\", \r\n\"tenantId\" : \"Data.tenantId\" \r\n}",
         "indexName": "fsm",
         "aggrQuery": "{\"aggs\":{\"AGGR\":{\"filter\":{\"bool\":{\"must_not\":[{\"term\":{\"Data.fsm.tenantId.keyword\":\"pb.testing\"}}]}},\"aggs\":{\"Applications Closed\":{\"filter\":{\"bool\":{\"must\":[{\"terms\":{\"Data.status.keyword\":[\"completed\"]}}]}},\"aggs\":{\"Count\":{\"value_count\":{\"field\":\"Data.@timestamp\"}}}}}}}}"
```
Comment (phanikiran-wtt-egov) [inline egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json:452]: Data.@timestamp should be changed to Data.fsm.@timestamp in thsi aggrQuery

Completed Status is CAPITAL Case COMPLETED and you can also consider application in status CITIZEN_FEEDBACK_PENDING also as completed, And not sure how Data.status.keyword will get the applicationStatus, it should be Data.fsm.applicationStatus

File: egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json [hunk 3]
```diff
@@ -527,7 +527,7 @@
       },
       {
         "module": "FSM",
-        "dateRefField": "Data.@timestamp",
+        "dateRefField": "Data.fsm.@timestamp",
         "requestQueryMap": "{\"wardId\" : \"Data.ward.name.keyword\", \r\n  \"district\" : \"Data.tenantData.city.districtCode\", \r\n\"tenantId\" : \"Data.tenantId\" \r\n}",
         "indexName": "fsm",
         "aggrQuery": "{\"aggs\":{\"AGGR\":{\"filter\":{\"bool\":{\"must_not\":[{\"term\":{\"Data.fsm.tenantId.keyword\":\"pb.testing\"}}]}},\"aggs\":{\"FSM\":{\"value_count\":{\"field\":\"Data.tenantId.keyword\"}}}}}}"
```
Comment (phanikiran-wtt-egov) [inline egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json:533]: In the value_count Data.fsm.tenantId should be used.

File: egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json [hunk 4]
```diff
@@ -575,7 +576,7 @@
       },
       {
         "module": "FSM",
-        "dateRefField": "Data.@timestamp",
+        "dateRefField": "Data.fsm.@timestamp",
         "requestQueryMap": "{\"wardId\" : \"Data.ward.name.keyword\", \r\n  \"district\" : \"Data.tenantData.city.districtCode\", \r\n\"tenantId\" : \"Data.fsm.tenantid.keyword\" \r\n}",
         "indexName": "fsm",
         "aggrQuery": "{\"aggs\":{\"AGGR\":{\"filter\":{\"bool\":{\"must_not\":[{\"term\":{\"Data.fsm.tenantid.keyword\":\"pb.testing\"}}]}},\"aggs\":{\"Total Application\":{\"date_histogram\":{\"field\":\"Data.@timestamp\",\"interval\":\"intervalvalue\"},\"aggs\":{\"Count\":{\"value_count\":{\"field\":\"Data.fsm.applicationNo.keyword\"}}}},\"Closed Application\":{\"date_histogram\":{\"field\":\"Data.@timestamp\",\"interval\":\"intervalvalue\"},\"aggs\":{\"Applications Closed\":{\"filter\":{\"bool\":{\"must\":[{\"terms\":{\"Data.status.keyword\":[\"completed\"]}}]}},\"aggs\":{\"Count\":{\"value_count\":{\"field\":\"Data.fsm.applicationNo.keyword\"}}}}}}}}}}"
```
Comment (phanikiran-wtt-egov) [inline egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json:582]: Data.@timestamp should be changed to Data.fsm.@timestamp in thsi aggrQuery

Completed Status is CAPITAL Case COMPLETED and you can also consider application in status CITIZEN_FEEDBACK_PENDING also as completed. And not sure how Data.status.keyword will get the applicationStatus, it should be Data.fsm.applicationStatus

**Review by phanikiran-wtt-egov — CHANGES_REQUESTED** [review]

**Review by shan-egov — APPROVED** [review]

**Review by sathishp-eGov — APPROVED** [review]

**Pull request closed without merging**